client/my-sites/email/form/mailboxes/types.ts

```diff
@@ -11,11 +11,19 @@ enum EmailProvider {
 interface MailboxFormField< Type > {
 	error: FieldError;
 	value: Type;
+	readonly typeName: string;
+	readonly required: boolean;
```

Comment:
We should probably add a prefix to this boolean variable for consistency purpose:

```suggestion
	readonly isRequired: boolean;
```